server/services/file_service.go

```
@@ -0,0 +1,110 @@
+func (fs *FileService) Unzip(args UnzipArgs) (err error) {
+	r, err := zip.OpenReader(args.ZipFile)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err := r.Close(); err != nil {
+			return
+		}
+	}()
+
+	os.MkdirAll(args.Destination, 0755)
```

golangci-lint (1.62.2)

Error return value of `os.MkdirAll` is not checked

(errcheck)